data/os/FreeBSD.yaml

```
@@ -7,5 +7,6 @@ nsd::service_name: 'nsd'
 nsd::owner: 'nsd'
 nsd::group: 'nsd'
 nsd::control_cmd: 'nsd-control'
+nsd::control_setup_cmd: 'nsd-control-setup'
```

On FreeBSD these are:
```suggestion
nsd::control_cmd: '/usr/local/sbin/nsd-control'
nsd::control_setup_cmd: '/usr/local/sbin/nsd-control-setup'
```